index.html

```diff
@@ -8,10 +8,40 @@
     <title>Vite App</title>
```

Comment:
You can change the title of the app to something more meaningful :)

Reply:
that is very true!